error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/MonoZipOfMonoVoidUsageTest.java

```diff
@@ -0,0 +1,110 @@
+package tech.picnic.errorprone.bugpatterns;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.jupiter.api.Test;
+
+final class MonoZipOfMonoVoidUsageTest {
+  @Test
+  void identification() {
+    CompilationTestHelper.newInstance(MonoZipOfMonoVoidUsage.class, getClass())
+        .expectErrorMessage(
+            "X",
+            m ->
```

Comment:
We don't need this setup :), using 

```
 "    // BUG: Diagnostic contains:",
```
Instead of the `matches: X` means exactly the same.

Reply:
I wanted to explicitly check on the error message though @rickie